aruna/api/storage/services/v1/project_service.proto

```diff
@@ -139,6 +139,20 @@ service ProjectService {
     };
   }
 
+
+  // GetAllUserPermissionsForProject
+  //
+  // Status: ALPHA
+  // 
+  // Get the user_permission of a specific user for the project.
+  rpc GetAllUserPermissionsForProject(GetAllUserPermissionsForProjectRequest)
+      returns (GetAllUserPermissionsForProjectResponse) {
+    option (google.api.http) = {
+      get : "/v1/project/{project_id}/users"
```

Comment:
```suggestion
      get : "/v1/project/{project_id}/get_users"
```

Just a suggestion, since the endpoint to request project permissions for a single user ends with `get_user`.